### PR TITLE
Support new delete

### PR DESCRIPTION
**Pull request by jt2594838**
see：
https://tower.im/projects/36de8571a0ff4833ae9d7f1c5c400c22/todos/83455db42c104746849565c5ca0a3eca/

### CODE REVIEW

File: src/main/java/cn/edu/tsinghua/tsfile/timeseries/readV2/reader/impl/SeriesChunkReaderWithFilterImpl.java [hunk 1]
```diff
@@ -33,6 +33,8 @@ public SeriesChunkReaderWithFilterImpl(InputStream seriesChunkInputStream, TSDat
 
     @Override
     public boolean pageSatisfied(PageHeader pageHeader) {
+        if(pageHeader.data_page_header.max_timestamp < getMaxTombstoneTime())
```
Comment (Beyyes) [inline src/main/java/cn/edu/tsinghua/tsfile/timeseries/readV2/reader/impl/SeriesChunkReaderWithFilterImpl.java]: The code can be formatted.

File: src/main/java/cn/edu/tsinghua/tsfile/file/metadata/TimeSeriesChunkMetaData.java [hunk 2]
```diff
@@ -222,4 +234,20 @@ public VInTimeSeriesChunkMetaData getVInTimeSeriesChunkMetaData() {
     public void setVInTimeSeriesChunkMetaData(VInTimeSeriesChunkMetaData vInTimeSeriesChunkMetaData) {
         this.vInTimeSeriesChunkMetaData = vInTimeSeriesChunkMetaData;
     }
+
+    public long getMaxTombstoneTime() {
```
Comment (Beyyes) [inline src/main/java/cn/edu/tsinghua/tsfile/file/metadata/TimeSeriesChunkMetaData.java]: Are these methods used?

File: src/main/java/cn/edu/tsinghua/tsfile/file/metadata/TimeSeriesChunkMetaData.java [hunk 3]
```diff
@@ -53,6 +53,18 @@
...
+     * The maximum time of the tombstones that take effect on this chunk. Only data with larger timestamps than this
+     * should be exposed to user.
+     */
+    private long maxTombstoneTime;
+
+    /**
+     * The time when the RowGroup of this chunk is closed. This will not be written out and will only be set when read together
+     * with its RowGroup during querying.
+     */
+    private long writtenTime;
```
Comment (Beyyes) [inline src/main/java/cn/edu/tsinghua/tsfile/file/metadata/TimeSeriesChunkMetaData.java]: What's the difference of variable 'writtenTime' between class TimeSeriesChunkMetaData and RowGroupMetaData? Maybe I haven't read the code of IoTDB... I can not understand the code currently.

File: src/main/java/cn/edu/tsinghua/tsfile/timeseries/readV2/reader/impl/SeriesChunkReaderWithFilterImpl.java (hunk 1 above)
Comment (jt2594838) [reply src/main/java/cn/edu/tsinghua/tsfile/timeseries/readV2/reader/impl/SeriesChunkReaderWithFilterImpl.java]: Done.

File: src/main/java/cn/edu/tsinghua/tsfile/file/metadata/TimeSeriesChunkMetaData.java (hunk 2 above)
Comment (jt2594838) [reply src/main/java/cn/edu/tsinghua/tsfile/file/metadata/TimeSeriesChunkMetaData.java]: Used only in IoTDB.

**Pull request merged**